### PR TITLE
<Link> refactoring

### PR DESCRIPTION
**Pull request by slowbackspace**
- switched hover effect (no underline by default, underline on hover)
- green link as default now

### CODE REVIEW

File: src/components/Link/index.js [hunk 1]
```diff
@@ -13,11 +13,13 @@ const A = styled.a`
 
     ${props => props.isGreen
         && css`
-            border-bottom: 1px solid ${colors.GREEN_PRIMARY};
+            text-decoration: underline;
+            text-decoration-color: ${colors.GREEN_PRIMARY};
```
Comment (keip) [inline src/components/Link/index.js]: text-decoration-color is unnecessary.
There should also be a hover efect with text-decoration: none; on line 40

Comment (slowbackspace) [reply src/components/Link/index.js]: let's take it a step further and make it underline by default.

**Comment by keip** [comment]
The problem now is that when you have a Grey link in text it's not visible.

tinygrab screen shot 01-03-2019 14 28 15

Maybe i would use the TEXT_PRIMARY color for the Grey link.
Or think about the underline as it was.

**Comment by slowbackspace** [comment]
> The problem now is that when you have a Grey link in text it's not visible.
> 
> tinygrab screen shot 01-03-2019 14 28 15
> 
> Maybe i would use the TEXT_PRIMARY color for the Grey link.
> Or think about the underline as it was.

We are going to leave it as it is as we discussed today. Default link is green. If developer chooses grey link lets hope he has some good reasons to do it (and there may be use cases when it is valid).

**Review by keip — APPROVED** [review]

**Pull request merged**